samples/unity-of-bugs/Assets/Scripts/SentryNativeBridge.m

```diff
@@ -0,0 +1,121 @@
+#import <Sentry/Sentry.h>
+
+void SentryNativeBridgeAddBreadcrumb(const char* timestamp, const char* message, const char* type, const char* category, int* level) {
```

Comment:
This file should move into package-dev/Plugins/iOS so it's inferred to be iOS only and gets bundled with the packake

Reply:
Agreed, it belongs somewhere else. 
If we put it in `Plugins/iOS` tho it ends up in `Libraries/io.sentry.unity/Plugins/iOS/` right next to Sentry.framework. Does it belong there?